webapp/i18n/es.json

```diff
@@ -695,7 +695,7 @@
   "channel_modal.group": "Grupo",
   "channel_modal.modalTitle": "Nuevo ",
   "channel_modal.name": "Nombre",
-  "channel_modal.nameEx": "Ej: \"Errores\", \"Mercadeo\", \"办公室恋情\"",
+  "channel_modal.nameEx": "Ej: \"Errores\", \"Mercadeo\", \"\"",
```

Comment:
Propose non-English include a multi-lingual example to make it clear multiple languages are supported.